src/modules/rlm_eap/types/rlm_eap_fast/eap_fast.c

```diff
@@ -905,18 +903,6 @@ static rlm_rcode_t CC_HINT(nonnull) process_reply( eap_handler_t *eap_session,
 	}
 
 
-	/*
-	 * Pack any tunnelled VPs and send them back
-	 * to the supplicant.
-	 */
-	if (tunnel_vps) {
-		RDEBUG("Sending tunneled reply attributes");
-		rdebug_pair_list(L_DBG_LVL_2, request, tunnel_vps, NULL);
-
-		eap_vp2fast(tls_session, tunnel_vps);
-		fr_pair_list_free(&tunnel_vps);
-	}
-
```

Comment:
Hmm, this looks pretty used to me... Why are you removing it? If it's because of a previous commit, can you reference that commit...